dvc/repo/experiments/__init__.py

```diff
@@ -79,6 +88,13 @@ def exp_dvc(self):
 
         return Repo(self.exp_dvc_dir)
 
+    @contextmanager
+    def chdir(self):
+        cwd = os.getcwd()
+        os.chdir(self.exp_dvc.root_dir)
+        yield self.exp_dvc.root_dir
+        os.chdir(cwd)
+
```

Comment:
Don't see it being used anywhere

Reply:
It's used in `experiments show`, but got removed in the last PR by mistake.

https://github.com/iterative/dvc/blob/11d906ee517e1980ce9884427a33b17253b1e4c2/dvc/repo/experiments/show.py#L61